### PR TITLE
[spike] chore: replace npm-shrinkwrap with package-lock

### PR DESCRIPTION
**Pull request by juliangieseke**
seems like github alerts integration only works with `package-lock.json` file

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -100,16 +99,3 @@ gulp.task("serve", function() {
     root: "./dist"
   });
 });
-
-// remove 'fsevents' from shrinkwrap, since it causes errors on non-Mac hosts
-// see https://github.com/npm/npm/issues/2679
-gulp.task("fixShrinkwrap", function(done) {
```
Comment (nLight) [inline gulpfile.js]: let's remove `build-shrinkwrap` from package.json

**Comment by GeorgiSTodorov** [comment]
File conflicts.

**Comment by DanielMSchmidt** [comment]
@juliangieseke ⚔️ Conflicts

**Comment by juliangieseke** [comment]
@GeorgiSTodorov @DanielMSchmidt @nLight branch updated

**Pull request merged**